test/AccountManager.test.ts

```diff
@@ -98,6 +98,42 @@ describe('AccountManager', function () {
       });
     });
 
+    describe('removeAccount()', function () {
+      let fakeAccounts: Array<Wallet>;
+
+      beforeEach(function () {
+        fakeAccounts = new Array<Wallet>(3).fill(Wallet.createRandom());
+
+        (accountManager as unknown as { _accounts: Wallet[] })._accounts =
+          fakeAccounts;
+      });
+
+      it('Should remove only account sent as parameter', function () {
+        accountManager.removeAccount(fakeAccounts[1]?.address as string);
+
+        expect(
+          accountManager.getAccounts().length === fakeAccounts.length - 1,
+          'Incorrect length'
+        );
```

Comment:
I know it's the same, but for readabiltiy I'd prefer to have something like
```suggestion
        expect(
          accountManager.getAccounts()
        ).to.be.equal(fakeAccounts.length - 1);

Reply:
Done.